index.html

```diff
@@ -137,7 +139,26 @@ <h3>Vanderbilt University | Jan 2019 - Mar 2020</h3>
 
     <!-- </section> -->
 
-
+    <experienceleftside>
```

Comment:
Added experience in senior year of high school - being President of a start up STEM education company.